### PR TITLE
[43] success page integration

### PR DESCRIPTION
**Pull request by oaedward**
# PR Description
Success Page Design and Integration

## Screenshots (if appropriate):
Screenshot 2023-05-14 at 14 28 48

### CODE REVIEW

**Comment by oaedward** [comment]
Lemme check
Your screenshot seems to be the old PR I made

**Review by akshaypuli — CHANGES_REQUESTED** [review]
@oaedward  Small changes but functionality works!

1. Placement of div to be centered
2. Removed of success page from dashboard

**Comment by oaedward** [comment]
To be clear @akshaypuli, only the dashboard link should be on the success page right?

**Comment by akshaypuli** [comment]
> To be clear @akshaypuli, only the dashboard link should be on the success page right?

Yes, I am updating the dashboard and they can create new records from there

**Comment by oaedward** [comment]
Update has been made

All the buttons have been removed now, leaving the "return to dashboard" only.

**Review by akshaypuli — DISMISSED** [review]
Looks Great!

File: src/client/web/src/components/BIOMD/BIOMD/BIOMD-UI/UI-Landing.vue [hunk 1]
```diff
@@ -40,8 +40,6 @@
 <script setup>
 import Card from "../BIOMD-UI/UI-Card.vue";
 
-import { ref } from "vue";
```
Comment (irushavm) [inline src/client/web/src/components/BIOMD/BIOMD/BIOMD-UI/UI-Landing.vue:43]: Why was `ref` removed here?

**Review by irushavm — APPROVED** [review]

**Review by irushavm — APPROVED** [review]

**Review by akshaypuli — APPROVED** [review]

File: src/client/web/src/components/BIOMD/BIOMD/BIOMD-UI/UI-Landing.vue (hunk 1 above)
Comment (oaedward) [reply src/client/web/src/components/BIOMD/BIOMD/BIOMD-UI/UI-Landing.vue]: ref was unused in that component @irushavm

**Pull request merged**